CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.23.8] - 2022-08-31
+### Changed
+- Update Cluster Prep Helm chart to support namespace label-based authentication.
+  [cyberark/conjur-authn-k8s-client#482](https://github.com/cyberark/conjur-authn-k8s-client/pull/482)
+
```

Comment:
everything here LGTM, we were able to test out the functionality as well and it looks great.